Formula/rpki-client.rb

```diff
@@ -0,0 +1,49 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://github.com/rpki-client/rpki-client-portable"
+  url "https://github.com/rpki-client/rpki-client-portable/archive/refs/tags/7.4.tar.gz"
+  sha256 "cacb56107b29baef67c0117bc53d726c95684ad27a6affe2a1527470c67f5e86"
+  license "ISC"
+
+  depends_on arch: :intel
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "libressl"
+  depends_on "libtool"
+  depends_on macos: :big_sur
```

Comment:
Why doesn't it work on earlier versions of macOS?

Reply:
I believed libressl was modern OS only. I just checked, it should work fine back to Mojave. I'll amend